packages/nextjs/app/api/builders/[builderAddress]/grants/route.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from "next/server";
+import { getAllGrantsForUser } from "~~/services/database/grants";
+
+export async function GET(_request: Request, { params }: { params: { builderAddress: string } }) {
+  try {
+    const builderAddress = params.builderAddress;
+    const grants = await getAllGrantsForUser(builderAddress);
+    return NextResponse.json(grants);
+  } catch (error) {
+    return NextResponse.json(
+      { error: "Internal Server Error" },
```

Comment:
Just update this to `error` key instead of `message` at 5fef2f9 🙈 since I saw we kind of following the convention of sending error in `error`  key for all APIs also I had the condition to handle it in the fetcher function : 

https://github.com/BuidlGuidl/grants.buidlguidl.com/blob/d8e4478cc195e4aefd000eaef0d4fbd39f7994ef/packages/nextjs/utils/swr.ts#L5